code/client/iconimporter

```diff
@@ -137,27 +143,26 @@ def find_items_to_check(repo_path, itemlist=None):
     return pkg_list
 
 
-def generate_pngs_from_munki_items(repo_path, force=False, itemlist=None):
+def generate_pngs_from_munki_items(repo, force=False, itemlist=None):
     '''Generate PNGs from either pkgs or disk images containing applications'''
-    itemlist = find_items_to_check(repo_path, itemlist=itemlist)
-    icons_dir = os.path.join(repo_path, u'icons')
-    if not os.path.exists(icons_dir):
-        os.mkdir(icons_dir)
+    itemlist = find_items_to_check(repo, itemlist=itemlist)
```

Comment:
the `find_items_to_check` function no longer exists since it was renamed to findItemsToCheck